module-audio/Audio/Profiles/ProfileRoutingHeadphones.hpp

```diff
@@ -11,30 +11,26 @@ namespace audio
-                          .bitWidth      = 16,
-                          .flags         = static_cast<uint32_t>(
-                                       audio::codec::Flags::InputLeft) | // microphone use left audio channel
-                                   static_cast<uint32_t>(audio::codec::Flags::OutputMono),
-                          .outputVolume = static_cast<float>(volume),
-                          .inputGain    = static_cast<float>(gain),
-                          .inputPath    = audio::codec::InputPath::Headphones,
-                          .outputPath   = audio::codec::OutputPath::Headphones,
-                          .filterCoefficients =
-                              {qfilter_CalculateCoeffs(
-                                   audio::equalizer::FilterType::FilterHighPass, 997.f, 44100, 0.701f, 0),
-                               qfilter_CalculateCoeffs(
-                                   audio::equalizer::FilterType::FilterLowPass, 4993.7f, 44100, 0.701f, 0),
-                               qfilter_CalculateCoeffs(
-                                   audio::equalizer::FilterType::FilterNone, 15975.7f, 44100, 0.701f, -10),
-                               qfilter_CalculateCoeffs(
-                                   audio::equalizer::FilterType::FilterNone, 200.4f, 44100, 0.701f, -10),
-                               qfilter_CalculateCoeffs(
-                                   audio::equalizer::FilterType::FilterNone, 1496.7f, 44100, 0.701f, -4)}},
-                      AudioDevice::Type::Audiocodec)
+            : Profile(
+                  "Routing Headset",
+                  Type::RoutingHeadphones,
+                  audio::codec::Configuration{
+                      .sampleRate_Hz = 16000,
+                      .bitWidth      = 16,
+                      .flags =
+                          static_cast<uint32_t>(audio::codec::Flags::InputLeft) | // microphone use left audio channel
+                          static_cast<uint32_t>(audio::codec::Flags::OutputMono),
+                      .outputVolume = static_cast<float>(volume),
+                      .inputGain    = static_cast<float>(gain),
+                      .inputPath    = audio::codec::InputPath::Headphones,
+                      .outputPath   = audio::codec::OutputPath::Headphones,
+                      .filterCoefficients =
+                          {qfilter_CalculateCoeffs(audio::equalizer::FilterType::HighPass, 997.f, 44100, 0.701f, 0),
+                           qfilter_CalculateCoeffs(audio::equalizer::FilterType::LowPass, 4993.7f, 44100, 0.701f, 0),
+                           qfilter_CalculateCoeffs(audio::equalizer::FilterType::None, 15975.7f, 44100, 0.701f, -10),
+                           qfilter_CalculateCoeffs(audio::equalizer::FilterType::None, 200.4f, 44100, 0.701f, -10),
+                           qfilter_CalculateCoeffs(audio::equalizer::FilterType::None, 1496.7f, 44100, 0.701f, -4)}},
+                  AudioDevice::Type::Audiocodec)
```

Comment:
Why headset equalizer settings are hardcoded and not in config files as well?

Reply:
good point! I thought that we don't mess up with EQ for headphones as the audio path should be as clean as possible - it's up to headphones speakers how they reproduce the sound, but I was wrong - there are some settings for headphones too. I'll move it to the file too.